operator/cmd/mesh/operator.go

```diff
@@ -15,15 +15,22 @@
 package mesh
 
 import (
+	"fmt"
+
 	"github.com/spf13/cobra"
+
+	"istio.io/istio/pkg/url"
 )
 
+var depNote = "Note: This command has been deprecated, and will be removed in the future. " +
```

Comment:
The current statement from the project is that it is discouraged, but is **not** slated for removal

Reply:
Should we gate this on #44814? I don't believe that issue came to consensus

Reply:
Thanks for the clarification. It should be too early to add this note before we reach a consensus.